.changeset/custom-edge-id-generator.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I'm positive this naming convention is wrong (not enough lobsters 🦞 ) but happy to rename